postcss/darkmode.js

```diff
@@ -0,0 +1,40 @@
+const indent = (text, level) => text.split('\n')
+    .map(line => ''.padStart(level, '\t') + line)
+    .join('\n')
+
+const renderRule = (rule, transform) => {
+    if (!rule.selector) {
+        console.log(rule.toString())
+        return ''
+    }
+    const selectors = rule.selector.split(',').map(s => s.trim()).map(transform)
+    return `${selectors.join(', ')} {
+${indent(rule.nodes.map(n => n.toString() + ';').join('\n'), 1)}
+}`
+}
+
+module.exports = (options = { forceGlobal: true }) => {
```

Comment:
Can we actually default this to false? It seems the most intuitive would be to assume it won't, and then change that flag explicitly in the svelte config.

Reply:
Yup, I went back and forwards on this a few times but happy to default it to false :smile: